.gitignore

```diff
@@ -75,8 +75,8 @@ build-iPhoneSimulator/
 *.log
 *.xml
 
-# allow the VZE Non-CR3 upload upload template
-!app/public/files/non_cr3_template.csv
+# allow the VZE Non-CR3 upload template
+!editor/public/files/non_cr3_template.csv
```

Comment:
oops, this file wasn't available in staging or prod! i never noticed the issue because i had the file locally